src/main/scala/PrivateHome/UI/repl.scala

```diff
@@ -25,52 +25,64 @@ import org.slf4j.LoggerFactory
 import java.io._
 
 
-
 object repl {
-  private val logger = LoggerFactory.getLogger(this.getClass)
   val socketPath = "/tmp/privatehome2.sock"
```

Comment:
Why is the socket named "privatehome2.sock"?

Reply:
Because of the old socketfile which was simply privatehome.sock

Reply:
This is fixed now